Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM ubuntu:latest
+RUN apt update
+# Avoid prompt for locale.
+RUN DEBIAN_FRONTEND="noninteractive" apt install -y tzdata
+RUN apt install -y git netcat wget
+RUN apt install -y openjdk-11-jdk maven
+# Used for waiting for services before starting Olog.
+RUN apt install -y wait-for-it
+RUN mkdir olog-es
+WORKDIR /olog-es
+COPY . .
+RUN mvn clean install -DskipTests=true -Pdeployable-jar
```

Comment:
Since you use Docker to build the application with Maven, i would recommend using Docker multi-stage build: https://docs.docker.com/develop/develop-images/multistage-build/
Otherwise you will have application source and build dependencies (like Maven) included in the resulting image. This means a bigger resulting image.